shared_model/schema/commands.proto

```diff
@@ -97,6 +98,7 @@ message RevokePermission {
 message SubtractAssetQuantity {
     string asset_id = 1;
     string amount = 2;
+    string title = 3;
```

Comment:
This param in previous versions of releases does not exist. So if we add the param we must be sure that this will not break old (but working) code. So please make the param optional.